editor/plugins/control_editor_plugin.cpp

```diff
@@ -959,6 +1006,33 @@ void ControlEditorToolbar::_selection_changed() {
 			container_h_picker->set_allowed_flags(allowed_all_flags);
 			container_v_picker->set_allowed_flags(allowed_all_flags);
 		}
+
+		// Update expand toggles.
+		int nb_valid_controls = 0;
+		int nb_h_expand = 0;
+		int nb_v_expand = 0;
+
+		List<Node *> selection = editor_selection->get_selected_node_list();
+		for (Node *E : selection) {
+			Control *control = Object::cast_to<Control>(E);
+			if (!control) {
+				continue;
+			}
+			if (!Object::cast_to<Container>(control->get_parent())) {
```

Comment:
Expand flag can be set outside containers (mostly for scene root that's going to be inside container later), so this check shouldn't be here I think.